### DISCUSSION TITLE
Minimized start option when run at windows start

### DISCUSSION THREAD

**Discussion by Evixone** [BlueBird67/BlackScreens#2] [Ideas]
Hi, there,
I love your application which allows me to lower the brightness on the fly with a keyboard shortcut at night.
But when I launch it at windows startup, the application opens and I have to click on the minimize button to hide it in the system tray.
Is it possible to add an option that would enable the minimized application to be launched in the system tray?
This would make the application the perfect version.
Thanks a lot

**Comment by BlueBird67** [comment]
Hi, thank you for your kind comment. I discover a new way of using this app from you, love that!

Could you tell me which Windows version you are using, and the number of screen you have?

For your enhancement question, yes I can do that of course. I will check on it, and come back to you with a new version.

**Reply by Evixone** [reply]
Yes, the application is light and really solves the problem of screens that don't have the CCI / IC function that lets you control brightness directly with Windows.
I've found another application to do the same thing, display fusion pro, but it's a gas factory that's not really light and the function doesn't respond quickly like blackscreens.
I have 2 screens connected, a monitor and a TV.
I have Windows 11 up to date.
Thanks a lot!

**Reply by BlueBird67** [reply]
Version 1.10 is ready with your option. Sadly, lightning strike injured my internet box yesterday. I must wait until end of week to have internet back again with a new box.Will let you know here when it's commited.

Sorry for this delay. ☺️

**Reply by Evixone** [reply]
No problem, amazing!

**Reply by BlueBird67** [reply]
Hi, happy to announce the release of [version 1.10](https://github.com/BlueBird67/BlackScreens/tree/1.10) with your option. Would love to have you test it.

**Reply by Evixone** [reply]
Good ! But My computer had a problem, I sent it for repair under warranty, so I should have it back in 10 days and I'll keep you posted, thanks!

**Comment by Evixone** [comment]
Very cool ! i have test it and it's good but there are the icon on the start bar when windows start 
<img width="452" alt="Capture d’écran 2023-07-28 201047" src="https://github.com/BlueBird67/BlackScreens/assets/132999101/bb7d03be-a0ef-40a5-bea7-a0f721ba41ac">

**Reply by BlueBird67** [reply]
It's not what you expected, that when the app start, it's start only with an icon on the start bar?

**Reply by Evixone** [reply]
Not in the start bar, in the system tray , now she appear in the system tray AND in the start bar.
For clean windows startup we need the app launch only in the system tray
<img width="274" alt="Capture d’écran 2023-07-28 211224" src="https://github.com/BlueBird67/BlackScreens/assets/132999101/0a377f0a-50a8-442d-98ba-8e99eace146b">
If you can, if not thank you anyway !

**Reply by Evixone** [reply]
And now there's a problem with the black screen. It doesn't go to the top, when I'm on google chrome the chrome windows go to the top and they're not under the black screen.

**Reply by BlueBird67** [reply]
Ok will check  on that.

**Reply by BlueBird67** [reply]
Ok I find this one fast. It's corrected on v1.11.
For the start bar, wasn't able to replicate the problem.

**Reply by Evixone** [reply]
You are on windows 11? in the 1.11 there are always the black screen under other windows, and the function "allow click through" doesn't work on first time , i need to disable and enable the function.

**Reply by BlueBird67** [reply]
Does all the windows are on top of Blackscreen, or just some of them? Yes I'm on W11.

**Reply by BlueBird67** [reply]
Ok I was able to reproduce. Very strange as it is working well in debug mode, but not with program alone. I need to dig more. Will keep you inform.

**Reply by BlueBird67** [reply]
Version 1.12 should normally fixed all your problems. Let me know if it's ok.

**Reply by Evixone** [reply]
Hello!
I just got my pc back from the repair shop.
the software works better, there's no longer the icon in the start bar and the black window is well above everything else, so there's been a big improvement! but there's still just one problem, when windows has started and so has the program, if I activate the black window with the keyboard shortcut, the "allow click trough" function doesn't work directly, I have to open the program, deactivate it and reactivate it once, then everything works perfectly.
